src/openexr.mk

```diff
@@ -19,50 +20,16 @@ define $(PKG)_UPDATE
 endef
 
 define $(PKG)_BUILD
-    # Update auto-stuff, except autoheader, because if fails...
-    cd '$(1)' && AUTOHEADER=true autoreconf -fi
-    # unpack and build a native version of ilmbase
-    cd '$(1)' && $(call UNPACK_PKG_ARCHIVE,ilmbase)
-    $(foreach PKG_PATCH,$(sort $(wildcard $(TOP_DIR)/src/ilmbase-*.patch)),
-        (cd '$(1)/$(ilmbase_SUBDIR)' && $(PATCH) -p1 -u) < $(PKG_PATCH))
-    echo 'echo $1' > '$(1)/$(ilmbase_SUBDIR)/config.sub'
-    cd '$(1)/$(ilmbase_SUBDIR)' && $(SHELL) ./configure \
-        --build="`config.guess`" \
-        --disable-shared \
-        --prefix='$(1)/ilmbase' \
-        --enable-threading=no \
-        --disable-posix-sem \
-        CONFIG_SHELL=$(SHELL) \
-        SHELL=$(SHELL)
-    $(MAKE) -C '$(1)/$(ilmbase_SUBDIR)' -j '$(JOBS)' install \
-        bin_PROGRAMS= sbin_PROGRAMS= noinst_PROGRAMS=
-    cd '$(1)' && ./configure \
-        $(MXE_CONFIGURE_OPTS) \
-        --disable-threading \
-        --disable-posix-sem \
-        --disable-ilmbasetest \
-        PKG_CONFIG='$(PREFIX)/bin/$(TARGET)-pkg-config' \
-        CXXFLAGS="-g -O2"
-    # build the code generator manually
-    cd '$(1)/IlmImf' && $(BUILD_CXX) -O2 \
-        -I'$(1)/ilmbase/include/OpenEXR' \
-        -L'$(1)/ilmbase/lib' \
-        b44ExpLogTable.cpp \
-        -lHalf \
-        -o b44ExpLogTable
-    '$(1)/IlmImf/b44ExpLogTable' > '$(1)/IlmImf/b44ExpLogTable.h'
-    cd '$(1)/IlmImf' && $(BUILD_CXX) -O2 \
-        -I'$(1)/config' -I. \
-        -I'$(1)/ilmbase/include/OpenEXR' \
-        -L'$(1)/ilmbase/lib' \
-        dwaLookups.cpp \
-        -lHalf -lIlmThread -lIex -lpthread \
-        -o dwaLookups
-    '$(1)/IlmImf/dwaLookups' > '$(1)/IlmImf/dwaLookups.h'
-    $(MAKE) -C '$(1)' -j '$(JOBS)' install bin_PROGRAMS= sbin_PROGRAMS= noinst_PROGRAMS=
+     cd '$(BUILD_DIR)' && $(TARGET)-cmake \
+    -DOPENEXR_CXX_STANDARD=11 \
+    -DOPENEXR_INSTALL_PKG_CONFIG=ON \
+    -DBUILD_TESTING=OFF \
+    "$(SOURCE_DIR)/OpenEXR"
+    $(MAKE) -C '$(BUILD_DIR)' -j '$(JOBS)'
+    $(MAKE) -C '$(BUILD_DIR)' -j 1 install
 
     '$(TARGET)-g++' \
-        -Wall -Wextra -std=gnu++0x \
+        -Wall -Wextra -std=c++$(if $($(PKG)_CXXSTD_14),14,11) \
```

Comment:
Where is `$(PKG)_CXXSTD_14` defined?

Reply:
It's an aborted idea (get gcc versions from gcc.mk, [ which wasn't always actually parsed when these were called ] and compare against version numbers known to provide good standards support for each version) but it was too messy and I forgot to remove it from that file; should be just c++11